test/test_framework_test.go

```diff
@@ -1112,8 +1112,7 @@ func TestImportBuiltinContracts(t *testing.T) {
             let address = bridgedAccount.address()
             destroy bridgedAccount
 
-            let expected: [UInt8; 20] = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 1]
-            assert(expected == address.bytes)
+            assert(address.bytes != [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
```

Comment:
I was not able to set a fixed expected value, because somehow the actual value is always different. Asked the EVM team about it: https://discordapp.com/channels/613813861610684416/1167476806333513800/1206772945436016640

Reply:
For the sake of testing:
```cadence
assert(address.bytes != [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
```
This is actually a better assertion that what I had, since it does not assert on implementation details that are liable to change. So I think we are good to go with this.